packages/liveblocks-node/src/client.ts

```diff
@@ -194,6 +273,518 @@ export class Liveblocks {
+  }
+
+  /**
+   * Updates specific properties of a room. It’s not necessary to provide the entire room’s information.
+   * Setting a property to `null` means to delete this property.
+   * @param roomId The id of the room to update.
+   * @param params.defaultAccesses (optional) The default accesses for the room.
+   * @param params.groupAccesses (optional) The group accesses for the room. Can contain a maximum of 100 entries. Key length has a limit of 40 characters.
+   * @param params.userAccesses (optional) The user accesses for the room. Can contain a maximum of 100 entries. Key length has a limit of 40 characters.
+   * @param params.metadata (optional) The metadata for the room. Supports upto a maximum of 50 entries. Key length has a limit of 40 characters. Value length has a limit of 256 characters.
+   * @returns The updated room.
+   */
+  public async updateRoom(
+    roomId: string,
+    params: {
+      defaultAccesses?: Permission[] | null;
+      groupAccesses?: Record<string, Permission[] | null>;
+      userAccesses?: Record<string, Permission[] | null>;
+      metadata?: Record<string, string | string[] | null>;
+    }
+  ): Promise<RoomInfo> {
+    const { defaultAccesses, groupAccesses, userAccesses, metadata } = params;
+
+    const res = await this.post(`/v2/rooms/${roomId}`, {
+      defaultAccesses,
+      groupAccesses,
+      userAccesses,
+      metadata,
+    });
+
+    if (!res.ok) {
+      const text = await res.text();
+      throw new ApiError(res.status, text);
+    }
+
+    return res.json() as Promise<RoomInfo>;
+  }
+
+  /**
+   * Deletes a room with the given id. A deleted room is no longer accessible from the API or the dashboard and it cannot be restored.
+   * @param roomId The id of the room to delete.
+   */
+  public async deleteRoom(roomId: string): Promise<void> {
+    const res = await this.delete(`/v2/rooms/${roomId}`);
+
+    if (!res.ok) {
+      const text = await res.text();
+      throw new ApiError(res.status, text);
+    }
+
+    return;
+  }
+
+  /**
+   * Returns a list of users currently present in the requested room. For better performance, we recommand to call this endpoint every 10 seconds maximum. Duplicates can happen if a user is in the requested room with multiple browser tabs opened.
+   * @param roomId The id of the room to get the users from.
+   * @returns A list of users currently present in the requested room.
+   */
+  public async getActiveUsers<T = unknown>(
+    roomId: string
+  ): Promise<RoomUser<T>[]> {
+    const res = await this.get(`/v2/rooms/${roomId}/active_users`);
+
+    if (!res.ok) {
+      const text = await res.text();
+      throw new ApiError(res.status, text);
+    }
+
+    return res.json() as Promise<RoomUser<T>[]>;
+  }
+
+  /**
+   * Boadcasts an event to a room without having to connect to it via the client from @liveblocks/client.
+   * @param roomId The id of the room to broadcast the event to.
+   */
+  public async broadcastMessage(
```

Comment:
Shouldn't this one be called `broadcastEvent`? Then it'd be in line with our other APIs.

Reply:
That is correct! Thanks for catching this. I've updated the function name to `broadcastEvent`